backend/src/index.js

```diff
@@ -0,0 +1,83 @@
+import { ApolloServer } from '@apollo/server'; // preserve-line
+import { startStandaloneServer } from '@apollo/server/standalone'; // preserve-line
+import { PrismaClient } from '@prisma/client';
+const prisma = new PrismaClient();
+// A schema is a collection of type definitions (hence "typeDefs")
+// that together define the "shape" of queries that are executed against
```

Comment:
Old compiled JavaScript files have made in to the commit. Delete `index.js` and `loginmutation.js` and ensure that they are not committed anymore.